kornia/losses/tversky.py

```diff
@@ -66,9 +72,23 @@ def tversky_loss(
     # compute softmax over the classes axis
     pred_soft: torch.Tensor = F.softmax(pred, dim=1)
 
+    # check if all pixels are valid
+    valid_idx = target != ignore_index
```

Comment:
maybe this can be wrapped into a reusable function across all the losses

Reply:
I pulled it out into a helper function, not sure where to place it though. Any suggestions appreciated.

Reply:
This sounds good